### PR TITLE
[ADD] account_voucher_no_auto_line (v8)

### PR DESCRIPTION
**Pull request by kittiu**
Port to v8

### CODE REVIEW

**Comment by elicoidal** [comment]
@kittiu   Thanks for your contribution to the OCA. 
Please help us in the initial commit and follow the principles: 
- Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
- We recommend to include a screenshot as well to help new comers (nice to have)
- Please use simple headers in all py files following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

**Comment by kittiu** [comment]
Thanks @elicoidal
I have follow the guideline. But CI test will still fail as there are other addons. How do I go about it?

File: account_voucher_no_auto_lines/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+.. branch is "8.0" for example
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```
Comment (elicoidal) [inline account_voucher_no_auto_lines/README.rst:31]: Please update accordingly

File: account_voucher_no_auto_lines/README.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "8.0" for example
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```
Comment (elicoidal) [inline account_voucher_no_auto_lines/README.rst]: Please update accordingly

File: account_voucher_no_auto_lines/__init__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline account_voucher_no_auto_lines/__init__.py:1]: Please use short headers

File: account_voucher_no_auto_lines/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(S)> <AUTHOR(S)>
```
Comment (elicoidal) [inline account_voucher_no_auto_lines/__openerp__.py]: Adapt to the current copyright

File: account_voucher_no_auto_lines/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Voucher No Auto Lines",
+    "summary": "No more refresh lines when change amount in Account Voucher",
```
Comment (elicoidal) [inline account_voucher_no_auto_lines/__openerp__.py]: When amount changes in Account Voucher, lines are not refreshed anymore

File: account_voucher_no_auto_lines/models/__init__.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline account_voucher_no_auto_lines/models/__init__.py:1]: header

**Comment by elicoidal** [comment]
Not sure (not expert): basically Flake8 issues in https://travis-ci.org/OCA/account-payment/jobs/109134597. @pedrobaeza any idea?

**Comment by kittiu** [comment]
@elicoidal, anything else that I can fix here, please let me know. Thanks!

**Comment by elicoidal** [comment]
@kittiu From my side no but I just review some wording and formalism.
Now you need to make CI happy ;)

**Comment by kittiu** [comment]
OK, @elicoidal I have fixed everything on this addon. Can I go ahead and change things other module also (account_vat_on_payment)?

**Comment by feketemihai** [comment]
@kittiu There is already a PR for vat_on_payment which is improving the module to be more configurable and the pylint issues are fixed there, see #54 .

**Comment by kittiu** [comment]
Ok. Want to learn about the process too :)

When this happen, how do we do? I should wait for that one to merge first, and I pull the latest again?

**Comment by feketemihai** [comment]
Since the errors doesn't relate with the actual module it can be merged even travis is failing..

**Comment by kittiu** [comment]
So, I guess it is ready now. Thank you.

**Comment by ghost** [comment]
@kittiu this is a dupplicate pull request :-1: instead please review https://github.com/OCA/account-payment/pull/56

**Comment by pedrobaeza** [comment]
As it's a duplicate, I close.

**Pull request closed without merging**